master/src/main/java/org/apache/celeborn/service/deploy/master/SlotsAllocator.java

```diff
@@ -219,7 +219,7 @@ private static StorageInfo getStorageInfo(
                 diskInfos[diskIndex].storageType(),
                 availableStorageTypes);
         diskIndex = (diskIndex + 1) % diskInfos.length;
-        workerDiskIndex.put(selectedWorker, (diskIndex + 1) % diskInfos.length);
+        workerDiskIndex.put(selectedWorker, diskIndex);
```

Comment:
is the change correct? `diskIndex` was updated before the second time calculation

Reply:
There should be no problem. StorageInfo has been constructed successfully. workerDiskIndex uses diskIndex to record the location of the next Disk.